packages/trpc/server/routers/viewer/teams/getUpgradeable.handler.ts

```diff
@@ -12,14 +12,32 @@ type GetUpgradeableOptions = {
 
 export const getUpgradeableHandler = async ({ ctx }: GetUpgradeableOptions) => {
   if (!IS_TEAM_BILLING_ENABLED) return [];
-  let { teams } = await prisma.user.findUniqueOrThrow({
-    where: { id: ctx.user.id },
-    include: { teams: { where: { role: MembershipRole.OWNER }, include: { team: true } } },
+
+  // Get all teams/orgs where the user is an owner
+  let teams = await prisma.membership.findMany({
+    where: {
+      user: {
+        id: ctx.user.id,
+      },
+      role: MembershipRole.OWNER,
+      team: {
+        parentId: null, // Since ORGS relay on their parent's subscription, we don't need to return them
+      },
+    },
+    include: {
+      team: {
+        include: {
+          children: true,
+        },
+      },
+    },
   });
+
   /** We only need to return teams that don't have a `subscriptionId` on their metadata */
   teams = teams.filter((m) => {
     const metadata = teamMetadataSchema.safeParse(m.team.metadata);
     if (metadata.success && metadata.data?.subscriptionId) return false;
+    if (m.team.children.length > 0) return false; // We also dont return ORGs as it will be handled in OrgUpgradeBanner
```

Comment:
We check children here so we can still charge if a "ORG" has lots of members but no teams